tests/integration/targets/mso_schema_template_anp_epg/tasks/main.yml

```diff
@@ -77,6 +77,7 @@
     schema: '{{ mso_schema | default("ansible_test") }}'
     template: Template 1
     vrf: VRF
+    preferred_group: true
```

Comment:
Why adding those preferred_group: true? Same for L96

Reply:
Looks like the template has Preferred Group enabled and not in the associated VRF policy. Because of that I get the error below:

`
EPG: ansible_test_3 in Schema: ansible_test, Template: Template2, has Preferred Group enabled and the associated VRF policy 'VRF3' in Template 'Template2' part of Schema 'ansible_test' has Preferred Group disabled
`